### PR TITLE
Create/Update profile form component created

### PR DESCRIPTION
**Pull request by SlowGti**
# TWIS-40/create-or-update-profile-form

## Link to Story

https://re-boot.atlassian.net/browse/TWIS-40

## Description

Create/Update profile form component created, also updated the Don't have an account component with our own button

## Screenshot(s) / GIF(s):

Mar-11-2024 13-31-49

## Type of change

- [ ] Bug fix
- [x] New feature
- [x] Dependency / Configuration update
- [ ] Documentation update

## Checklist:

- [x] My code follows the project's style guidelines
- [x] I have created a Storybook story for my component(s)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/app/_components/CreateUpdateProfileForm/CreateUpdateProfileForm.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+          fullWidth
+          value={username}
+          onChange={(e) => setUsername(e.target.value.slice(0, 15))}
+          variant="outlined"
+          inputProps={{ maxLength: 15 }}
+        />
+        <TextField
+          label="Email"
+          fullWidth
+          value="muzappar.erkin0122@gmail.com"
```
Comment (Muradil-Erkin) [inline src/app/_components/CreateUpdateProfileForm/CreateUpdateProfileForm.tsx:73]: email should be a required prop

File: src/app/_components/CreateUpdateProfileForm/CreateUpdateProfileForm.tsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
+import { useState } from "react";
+import { Box, TextField } from "@mui/material";
+import { Button } from "../Button";
+
+interface CreateUpdateProfileFormProps {
+  onCancel: () => void;
+  onDone: () => void;
```
Comment (Muradil-Erkin) [inline src/app/_components/CreateUpdateProfileForm/CreateUpdateProfileForm.tsx:7]: onSave?

**Review by manzire — APPROVED** [review]
image

Teasted all  . looks great.
manzire--QA team

**Review by ShinK8245 — APPROVED** [review]
LGTM

File: src/app/_components/CreateUpdateProfileForm/CreateUpdateProfileForm.tsx [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+  onCancel,
+  onSave,
+}: CreateUpdateProfileFormProps): JSX.Element => {
+  const [name, setName] = useState("");
+  const [username, setUsername] = useState("");
+  const [bio, setBio] = useState("");
+
+  const handleCancel = () => {
+    // Handle cancel action
+    console.log("Action Canceled");
```
Comment (Jack-Vu) [inline src/app/_components/CreateUpdateProfileForm/CreateUpdateProfileForm.tsx:20]: Should run these logs on the story and keep the console logs out of the actual component incase we forget to get rid of them later. Other then that looks good.

**Review by Jack-Vu — CHANGES_REQUESTED** [review]

File: src/app/_components/CreateUpdateProfileForm/CreateUpdateProfileForm.tsx (hunk 3 above)
Comment (SlowGti) [reply src/app/_components/CreateUpdateProfileForm/CreateUpdateProfileForm.tsx]: Removed console logs inside the components and moved it into stories

**Review by Jack-Vu — APPROVED** [review]
LGTM!

**Review by Muradil-Erkin — APPROVED** [review]

**Pull request merged**